packages/strapi-plugin-content-manager/admin/src/translations/it.json

```diff
@@ -81,9 +91,12 @@
   "form.Input.label.inputDescription": "Questo valore sovrascrive l'etichetta mostrata nell'intestazione della tabella",
   "form.Input.pageEntries": "Righe per pagina",
   "form.Input.pageEntries.inputDescription": "Attenzione: Puoi sovrascrivere questo valore nella pagina delle impostazioni del Tipo di Contenuto",
+  "form.Input.placeholder": "Segnaposto",
+  "form.Input.placeholder.placeholder": "Il mio fantastico valore",
```

Comment:
Are you sure about `fantastico` ?

Reply:
Sorry for my absence but i've a lot of work and i haven't no time to translate.
 fantastico is not the tight translation. It's more correct to use "eccezionale".